Documentation/api_v3.md

```diff
@@ -1,6 +1,33 @@
 # etcd3 API
 
-TODO: API doc
+TODO: finish API doc
+
+## Key-Value API
+
+Key-Value API is used to manipulate key-value pairs stored inside etcd. The key-value API is defined as a gRPC service[kv-service]. The Key-Value pair is defined as a structured data in [protobuf format][kv-proto].
+
+### Key-Value Pair
+
+Key-Value pair is the smallest unit that Key-Value API can manipulate. Each key-value pair has a few fields listed below.
```

Comment:
"A key-value pair is the smallest unit that the key-value API can manipulate. Each key-value pair has a number of fields:"